.pre-commit-config.yaml

```diff
@@ -24,4 +24,4 @@ repos:
       name: flake8 __init__.py files
       files: "__init__.py"
       # ignore unused import error in __init__.py files
-      args: ["--ignore=F401,E203"]
+      args: ["--ignore=F401,E203", --config, setup.cfg]
```

Comment:
Isn't this in master already: 4e001df7f3759389a9c84a6bcab392eca34c50cf?

Reply:
Only for non-init files. This adds it to the init.py target.